SECURITY-2.0-PLAN.md

```
@@ -288,6 +289,69 @@ artisanpack-ui/
+        'role'       => \ArtisanPackUI\Rbac\Models\Role::class,
+        'permission' => \ArtisanPackUI\Rbac\Models\Permission::class,
+    ],
+];
+```
+
+#### CMS Framework Integration
+
+The `artisanpack-ui/cms-framework` package currently has its own RBAC implementation (`Role`, `Permission` models, `HasRolesAndPermissions` trait, migrations, managers, policies, and API controllers). Once this RBAC package is available, the CMS framework should adopt it as a dependency to eliminate duplication.
+
+**What RBAC owns (CMS framework removes)**:
+| Concern | RBAC Package | CMS Framework |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)